apps/bemo-worker/src/worker.ts

```diff
@@ -11,16 +11,16 @@ import {
 	urlMetadataQueryValidator,
 } from '@tldraw/worker-shared'
 import { WorkerEntrypoint } from 'cloudflare:workers'
-import { Router, createCors } from 'itty-router'
+import { Router, cors } from 'itty-router'
 import { Environment } from './types'
 
 export { BemoDO } from './BemoDO'
 
-const cors = createCors({ origins: ['*'] })
+const { preflight, corsify } = cors({ origin: '*' })
 
 export default class Worker extends WorkerEntrypoint<Environment> {
 	private readonly router = Router()
-		.all('*', cors.preflight)
+		.all('*', preflight)
```

Comment:
it's the same behaviour i just couldn't be bothered to make the change to AutoRouter since it's not required and would have been a bigger diff. we can follow up after this PR